diff --git a/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js b/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
index 79c2168..3e60085 100644
--- a/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
+++ b/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
@@ -57,9 +57,9 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }) => {
     logos: { menu },
   } = useConfigurations();
   const [condensed, setCondensed] = usePersistentState('navbar-condensed', false);
-  const { userDisplayName } = useAppInfos();
+  const { userDisplayName, autoReload } = useAppInfos();
   const { formatMessage } = useIntl();
-
+  const hideLinksIds = ['/plugins/content-type-builder', '/plugins/seo', '/plugins/documentation'];
   const initials = userDisplayName
     .split(' ')
     .map(name => name.substring(0, 1))
@@ -116,12 +116,9 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }) => {
 
         {pluginsSectionLinks.length > 0 ? (
           <NavSection
-            label={formatMessage({
-              id: 'app.components.LeftMenu.plugins',
-              defaultMessage: 'Plugins',
-            })}
+            label={"Pluginy"}
           >
-            {pluginsSectionLinks.map(link => {
+             {pluginsSectionLinks.filter(l => hideLinksIds.indexOf(l.to) === -1 || autoReload).map(link => {
               const Icon = link.icon;
 
               return (
@@ -135,10 +132,7 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }) => {
 
         {generalSectionLinks.length > 0 ? (
           <NavSection
-            label={formatMessage({
-              id: 'app.components.LeftMenu.general',
-              defaultMessage: 'General',
-            })}
+            label={"Všeobecné"}
           >
             {generalSectionLinks.map(link => {
               const LinkIcon = link.icon;
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js
index a0d0fda..14a5659 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js
@@ -4,9 +4,8 @@ import { useIntl } from 'react-intl';
 import { ContentBox, useTracking } from '@strapi/helper-plugin';
 import { Stack } from '@strapi/design-system/Stack';
 import InformationSquare from '@strapi/icons/InformationSquare';
-import CodeSquare from '@strapi/icons/CodeSquare';
-import PlaySquare from '@strapi/icons/PlaySquare';
-import FeatherSquare from '@strapi/icons/FeatherSquare';
+import ShoppingCart from '@strapi/icons/ShoppingCart';
+import Earth from '@strapi/icons/Earth';
 
 const BlockLink = styled.a`
   text-decoration: none;
@@ -23,14 +22,14 @@ const ContentBlocks = () => {
   return (
     <Stack spacing={5}>
       <BlockLink
-        href="https://strapi.io/resource-center"
+        href="https://docs.google.com/document/d/1GK9UpfG6kHYclm1basHQtDhoPTdHNXWUo2KzaRBSah0/edit"
         target="_blank"
         rel="noopener noreferrer nofollow"
         onClick={() => handleClick('didClickonReadTheDocumentationSection')}
       >
         <ContentBox
           title={formatMessage({
-            id: 'global.documentation',
+            id: 'app.components.BlockLink.documentation',
             defaultMessage: 'Documentation',
           })}
           subtitle={formatMessage({
@@ -42,45 +41,26 @@ const ContentBlocks = () => {
         />
       </BlockLink>
       <BlockLink
-        href="https://strapi.io/starters"
-        target="_blank"
-        rel="noopener noreferrer nofollow"
-        onClick={() => handleClick('didClickonCodeExampleSection')}
-      >
-        <ContentBox
-          title={formatMessage({
-            id: 'app.components.BlockLink.code',
-            defaultMessage: 'Code example',
-          })}
-          subtitle={formatMessage({
-            id: 'app.components.BlockLink.code.content',
-            defaultMessage: 'Learn by using ready-made starters for your projects.',
-          })}
-          icon={<CodeSquare />}
-          iconBackground="warning100"
-        />
-      </BlockLink>
-      <BlockLink
-        href="https://strapi.io/blog/categories/tutorials"
-        target="_blank"
-        rel="noopener noreferrer nofollow"
-        onClick={() => handleClick('didClickonTutorialSection')}
+          href="https://www.sklinet.com/"
+          target="_blank"
+          rel="noopener noreferrer nofollow"
+          onClick={() => handleClick('didClickonTutorialSection')}
       >
         <ContentBox
           title={formatMessage({
             id: 'app.components.BlockLink.tutorial',
-            defaultMessage: 'Tutorials',
-          })}
-          subtitle={formatMessage({
-            id: 'app.components.BlockLink.tutorial.content',
-            defaultMessage: 'Follow step-by-step instructions to use and customize Strapi.',
-          })}
-          icon={<PlaySquare />}
-          iconBackground="secondary100"
+              defaultMessage: 'Tutorials',
+            })}
+            subtitle={formatMessage({
+                id: 'app.components.BlockLink.tutorial.content',
+                defaultMessage: 'Follow step-by-step instructions to use and customize Strapi.',
+            })}
+            icon={<Earth />}
+            iconBackground="secondary100"
         />
       </BlockLink>
-      <BlockLink
-        href="https://strapi.io/blog"
+      {/* <BlockLink
+        href="https://www.sklinet.com/"
         target="_blank"
         rel="noopener noreferrer nofollow"
         onClick={() => handleClick('didClickonBlogSection')}
@@ -94,10 +74,10 @@ const ContentBlocks = () => {
             id: 'app.components.BlockLink.blog.content',
             defaultMessage: 'Read the latest news about Strapi and the ecosystem.',
           })}
-          icon={<FeatherSquare />}
+          icon={<ShoppingCart />}
           iconBackground="alternative100"
         />
-      </BlockLink>
+      </BlockLink> */}
     </Stack>
   );
 };
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/SocialLinks.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/SocialLinks.js
index e5cb43b..ac21c10 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/SocialLinks.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/SocialLinks.js
@@ -3,197 +3,150 @@
  * SocialLink
  */
 
-import React from 'react';
-import styled from 'styled-components';
-import { useIntl } from 'react-intl';
-import { Typography } from '@strapi/design-system/Typography';
-import { Box } from '@strapi/design-system/Box';
-import { Stack } from '@strapi/design-system/Stack';
-import { Grid, GridItem } from '@strapi/design-system/Grid';
-import { LinkButton } from '@strapi/design-system/v2/LinkButton';
-import { Link } from '@strapi/design-system/v2/Link';
-import ExternalLink from '@strapi/icons/ExternalLink';
-import Github from '@strapi/icons/Github';
-import Discord from '@strapi/icons/Discord';
-import Reddit from '@strapi/icons/Reddit';
-import Strapi from '@strapi/icons/Strapi';
-import Twitter from '@strapi/icons/Twitter';
-import Discourse from '@strapi/icons/Discourse';
+ import React from 'react';
+ import styled from 'styled-components';
+ import { useIntl } from 'react-intl';
+ import { Typography } from '@strapi/design-system/Typography';
+ import { Box } from '@strapi/design-system/Box';
+ import { Stack } from '@strapi/design-system/Stack';
+ import { Grid, GridItem } from '@strapi/design-system/Grid';
+ import { LinkButton } from '@strapi/design-system/v2/LinkButton';
+ import Facebook from '@strapi/icons/Facebook';
+ import Strapi from '@strapi/icons/Strapi';
+ import Earth from '@strapi/icons/Earth';
+ import Pin from "@strapi/icons/Pin";
+ import Landscape from "@strapi/icons/Landscape";
+ import Play from "@strapi/icons/Play";
 
-const StyledDiscord = styled(Discord)`
-  path {
-    fill: #7289da !important;
-  }
-`;
 
-const StyledReddit = styled(Reddit)`
-  > path:first-child {
-    fill: #ff4500;
-  }
-`;
-const StyledStrapi = styled(Strapi)`
-  > path:first-child {
-    fill: #8e75ff;
-  }
-  > path:nth-child(2) {
-    fill: #8e75ff;
-  }
-  > path:nth-child(3) {
-    fill: #8e75ff;
-  }
-`;
+ const StyledStrapi = styled(Strapi)`
+   > path:first-child {
+     fill: #8e75ff;
+   }
+   > path:nth-child(2) {
+     fill: #8e75ff;
+   }
+   > path:nth-child(3) {
+     fill: #8e75ff;
+   }
+ `;
 
-const StyledTwitter = styled(Twitter)`
-  path {
-    fill: #1da1f2 !important;
-  }
-`;
+ const StyledEarth = styled(Earth)`
+ `;
+ const StyledPin = styled(Pin)`
+ `;
+ const StyledLandscape = styled(Landscape)`
+ `;
+ const StyledPlay = styled(Play)`
+ `;
 
-const StyledDiscourse = styled(Discourse)`
-  > path:first-child {
-    fill: #231f20;
-  }
-  > path:nth-child(2) {
-    fill: #fff9ae;
-  }
-  > path:nth-child(3) {
-    fill: #00aeef;
-  }
-  > path:nth-child(4) {
-    fill: #00a94f;
-  }
-  > path:nth-child(5) {
-    fill: #f15d22;
-  }
-  > path:nth-child(6) {
-    fill: #e31b23;
-  }
-`;
 
-const socialLinks = [
-  {
-    name: 'Github',
-    link: 'https://github.com/strapi/strapi/',
-    icon: <Github fill="#7289DA" />,
-    alt: 'github',
-  },
-  {
-    name: 'Discord',
-    link: 'https://slack.strapi.io/',
-    icon: <StyledDiscord />,
-    alt: 'discord',
-  },
-  {
-    name: 'Reddit',
-    link: 'https://www.reddit.com/r/Strapi/',
-    icon: <StyledReddit />,
-    alt: 'reddit',
-  },
-  {
-    name: 'Twitter',
-    link: 'https://twitter.com/strapijs',
-    icon: <StyledTwitter />,
-    alt: 'twitter',
-  },
-  {
-    name: 'Forum',
-    link: 'https://forum.strapi.io',
-    icon: <StyledDiscourse />,
-    alt: 'forum',
-  },
-  {
-    name: 'Blog',
-    link: 'https://strapi.io/blog?utm_source=referral&utm_medium=admin&utm_campaign=career%20page',
-    icon: <StyledStrapi />,
-    alt: 'blog',
-  },
-  {
-    name: 'We are hiring!',
-    link: 'https://strapi.io/careers?utm_source=referral&utm_medium=admin&utm_campaign=blog',
-    icon: <StyledStrapi />,
-    alt: 'career',
-  },
-];
+ const socialLinks = [
+   {
+     name: 'Facebook',
+     link: 'https://www.facebook.com/sklinet/',
+     icon: <Facebook fill="#7289DA" />,
+     alt: 'facebook',
+   },
+//    {
+//      name: 'Youtube',
+//      link: 'https://www.youtube.com/channel/UC0pZ62TY4wb1H1HqGn4QrKA',
+//      icon: <StyledPlay />,
+//      alt: 'youtube',
+//    },
+   {
+     name: 'Instagram',
+     link: 'https://www.instagram.com/sklinet_sk/',
+     icon: <StyledLandscape />,
+     alt: 'instagram',
+   },
+//    {
+//      name: 'Pinterest',
+//      link: 'https://cz.pinterest.com/whoopdedoocz/',
+//      icon: <StyledPin />,
+//      alt: 'pinterest',
+//    },
+   {
+     name: 'Web',
+     link: 'https://www.sklinet.com/',
+     icon: <StyledEarth />,
+     alt: 'web',
+   },
+ ];
 
-const LinkCustom = styled(LinkButton)`
-  display: flex;
-  align-items: center;
-  border: none;
+ const LinkCustom = styled(LinkButton)`
+   display: flex;
+   align-items: center;
+   border: none;
 
-  svg {
-    width: ${({ theme }) => theme.spaces[6]};
-    height: ${({ theme }) => theme.spaces[6]};
-  }
+   svg {
+     width: ${({ theme }) => theme.spaces[6]};
+     height: ${({ theme }) => theme.spaces[6]};
+   }
 
-  span {
-    word-break: keep-all;
-  }
-`;
+   span {
+     word-break: keep-all;
+   }
+ `;
 
-const GridGap = styled(Grid)`
-  row-gap: ${({ theme }) => theme.spaces[2]};
-  column-gap: ${({ theme }) => theme.spaces[4]};
-`;
+ const GridGap = styled(Grid)`
+   row-gap: ${({ theme }) => theme.spaces[2]};
+   column-gap: ${({ theme }) => theme.spaces[4]};
+ `;
 
-const SocialLinks = () => {
-  const { formatMessage } = useIntl();
+ const SocialLinks = () => {
+   const { formatMessage } = useIntl();
 
-  return (
-    <Box
-      as="aside"
-      aria-labelledby="join-the-community"
-      background="neutral0"
-      hasRadius
-      paddingRight={5}
-      paddingLeft={5}
-      paddingTop={6}
-      paddingBottom={6}
-      shadow="tableShadow"
-    >
-      <Box paddingBottom={7}>
-        <Stack spacing={5}>
-          <Stack spacing={3}>
-            <Typography variant="delta" as="h2" id="join-the-community">
-              {formatMessage({
-                id: 'app.components.HomePage.community',
-                defaultMessage: 'Join the community',
-              })}
-            </Typography>
-            <Typography textColor="neutral600">
-              {formatMessage({
-                id: 'app.components.HomePage.community.content',
-                defaultMessage:
-                  'Discuss with team members, contributors and developers on different channels',
-              })}
-            </Typography>
-          </Stack>
-          <Link href="https://feedback.strapi.io/" isExternal endIcon={<ExternalLink />}>
-            {formatMessage({
-              id: 'app.components.HomePage.roadmap',
-              defaultMessage: 'See our road map',
-            })}
-          </Link>
-        </Stack>
-      </Box>
-      <GridGap>
-        {socialLinks.map(socialLink => {
-          return (
-            <GridItem col={6} s={12} key={socialLink.name}>
-              <LinkCustom
-                size="L"
-                startIcon={socialLink.icon}
-                variant="tertiary"
-                href={socialLink.link}
-                isExternal
-              >
-                {socialLink.name}
-              </LinkCustom>
-            </GridItem>
-          );
-        })}
-      </GridGap>
-    </Box>
-  );
-};
+   return (
+     <Box
+       as="aside"
+       aria-labelledby="join-the-community"
+       background="neutral0"
+       hasRadius
+       paddingRight={5}
+       paddingLeft={5}
+       paddingTop={6}
+       paddingBottom={6}
+       shadow="tableShadow"
+     >
+       <Box paddingBottom={7}>
+         <Stack spacing={5}>
+           <Stack spacing={3}>
+             <Typography variant="delta" as="h2" id="join-the-community">
+               {formatMessage({
+                 id: 'app.components.HomePage.community',
+                 defaultMessage: 'Join the community',
+               })}
+             </Typography>
+             <Typography textColor="neutral600">
+               {formatMessage({
+                 id: 'app.components.HomePage.community.content',
+                 defaultMessage:
+                   'Discuss with team members, contributors and developers on different channels',
+               })}
+             </Typography>
+           </Stack>
+         </Stack>
+       </Box>
+       <GridGap>
+         {socialLinks.map(socialLink => {
+           return (
+             <GridItem col={6} s={12} key={socialLink.name}>
+               <LinkCustom
+                 size="L"
+                 startIcon={socialLink.icon}
+                 variant="tertiary"
+                 href={socialLink.link}
+                 isExternal
+               >
+                 {socialLink.name}
+               </LinkCustom>
+             </GridItem>
+           );
+         })}
+       </GridGap>
+     </Box>
+   );
+ };
 
-export default SocialLinks;
+ export default SocialLinks;
