diff --git a/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js b/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
index 752f3a6..b6e6281 100644
--- a/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
+++ b/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
@@ -57,10 +57,11 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }) => {
     logos: { menu },
   } = useConfigurations();
   const [condensed, setCondensed] = usePersistentState('navbar-condensed', false);
-  const { userDisplayName } = useAppInfos();
+  const { userDisplayName, autoReload } = useAppInfos();
   const { formatMessage } = useIntl();
   const { trackUsage } = useTracking();
   const { pathname } = useLocation();
+  const hideLinksIds = ['/plugins/content-type-builder', '/plugins/seo', '/plugins/documentation'];
 
   const initials = userDisplayName
     .split(' ')
@@ -127,12 +128,9 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }) => {
 
         {pluginsSectionLinks.length > 0 ? (
           <NavSection
-            label={formatMessage({
-              id: 'app.components.LeftMenu.plugins',
-              defaultMessage: 'Plugins',
-            })}
+            label={"Pluginy"}
           >
-            {pluginsSectionLinks.map((link) => {
+            {pluginsSectionLinks.filter(l => hideLinksIds.indexOf(l.to) === -1 || autoReload).map(link => {
               const Icon = link.icon;
 
               return (
@@ -152,10 +150,7 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }) => {
 
         {generalSectionLinks.length > 0 ? (
           <NavSection
-            label={formatMessage({
-              id: 'app.components.LeftMenu.general',
-              defaultMessage: 'General',
-            })}
+            label={"Všeobecné"}
           >
             {generalSectionLinks.map((link) => {
               const LinkIcon = link.icon;
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js
index 9feb7bb..cc96b96 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js
@@ -4,9 +4,7 @@ import { useIntl } from 'react-intl';
 import { ContentBox, useTracking } from '@strapi/helper-plugin';
 import { Stack } from '@strapi/design-system/Stack';
 import InformationSquare from '@strapi/icons/InformationSquare';
-import CodeSquare from '@strapi/icons/CodeSquare';
-import PlaySquare from '@strapi/icons/PlaySquare';
-import FeatherSquare from '@strapi/icons/FeatherSquare';
+import Earth from '@strapi/icons/Earth';
 
 const BlockLink = styled.a`
   text-decoration: none;
@@ -23,14 +21,14 @@ const ContentBlocks = () => {
   return (
     <Stack spacing={5}>
       <BlockLink
-        href="https://strapi.io/resource-center"
+        href="https://docs.google.com/document/d/1GK9UpfG6kHYclm1basHQtDhoPTdHNXWUo2KzaRBSah0/edit"
         target="_blank"
         rel="noopener noreferrer nofollow"
         onClick={() => handleClick('didClickonReadTheDocumentationSection')}
       >
         <ContentBox
           title={formatMessage({
-            id: 'global.documentation',
+            id: 'app.components.BlockLink.documentation',
             defaultMessage: 'Documentation',
           })}
           subtitle={formatMessage({
@@ -42,60 +40,22 @@ const ContentBlocks = () => {
         />
       </BlockLink>
       <BlockLink
-        href="https://strapi.io/starters"
-        target="_blank"
-        rel="noopener noreferrer nofollow"
-        onClick={() => handleClick('didClickonCodeExampleSection')}
-      >
-        <ContentBox
-          title={formatMessage({
-            id: 'app.components.BlockLink.code',
-            defaultMessage: 'Code example',
-          })}
-          subtitle={formatMessage({
-            id: 'app.components.BlockLink.code.content',
-            defaultMessage: 'Learn by using ready-made starters for your projects.',
-          })}
-          icon={<CodeSquare />}
-          iconBackground="warning100"
-        />
-      </BlockLink>
-      <BlockLink
-        href="https://strapi.io/blog/categories/tutorials"
-        target="_blank"
-        rel="noopener noreferrer nofollow"
-        onClick={() => handleClick('didClickonTutorialSection')}
+          href="https://www.sklinet.com/"
+          target="_blank"
+          rel="noopener noreferrer nofollow"
+          onClick={() => handleClick('didClickonTutorialSection')}
       >
         <ContentBox
           title={formatMessage({
             id: 'app.components.BlockLink.tutorial',
             defaultMessage: 'Tutorials',
-          })}
+            })}
           subtitle={formatMessage({
             id: 'app.components.BlockLink.tutorial.content',
             defaultMessage: 'Follow step-by-step instructions to use and customize Strapi.',
-          })}
-          icon={<PlaySquare />}
-          iconBackground="secondary100"
-        />
-      </BlockLink>
-      <BlockLink
-        href="https://strapi.io/blog"
-        target="_blank"
-        rel="noopener noreferrer nofollow"
-        onClick={() => handleClick('didClickonBlogSection')}
-      >
-        <ContentBox
-          title={formatMessage({
-            id: 'app.components.BlockLink.blog',
-            defaultMessage: 'Blog',
-          })}
-          subtitle={formatMessage({
-            id: 'app.components.BlockLink.blog.content',
-            defaultMessage: 'Read the latest news about Strapi and the ecosystem.',
-          })}
-          icon={<FeatherSquare />}
-          iconBackground="alternative100"
+            })}
+            icon={<Earth />}
+            iconBackground="secondary100"
         />
       </BlockLink>
     </Stack>
