diff --git a/node_modules/strapi-plugin-slugify/server/graphql/types.js b/node_modules/strapi-plugin-slugify/server/graphql/types.js
index 0aeef95..f9505a9 100644
--- a/node_modules/strapi-plugin-slugify/server/graphql/types.js
+++ b/node_modules/strapi-plugin-slugify/server/graphql/types.js
@@ -48,10 +48,11 @@ const getCustomTypes = (strapi, nexus) => {
 						modelName: nexus.stringArg('The model name of the content type'),
 						slug: nexus.stringArg('The slug to query for'),
 						publicationState: nexus.stringArg('The publication state of the entry'),
+                        locale: nexus.stringArg('Language')
 					},
 					resolve: async (_parent, args, ctx) => {
 						const { models } = getPluginService(strapi, 'settingsService').get();
-						const { modelName, slug, publicationState } = args;
+						const { modelName, slug, publicationState, locale } = args;
 						const { auth } = ctx.state;
 
 						isValidFindSlugParams({
@@ -72,6 +73,10 @@ const getCustomTypes = (strapi, nexus) => {
 							},
 						};
 
+                        if (locale) {
+                            query.locale = locale;
+                        }
+
 						// only return published entries by default if content type has draftAndPublish enabled
 						if (_.get(contentType, ['options', 'draftAndPublish'], false)) {
 							query.publicationState = publicationState || 'live';
